docs/en/userguide/01_User_manual/06_Remember_trusted_devices.md

```diff
@@ -23,10 +21,11 @@ When logging in with the **Keep me signed in** option selected, a time-based coo
 
 In addition to remembering the device, your username and password will also be remembered.
 
-<div class="notice" markdown='1'>
-It is important that you only choose to use this option on a device you know to be secure. A secure device may be one that requires you to log in before being able to access the Internet, for example one within an office network, or a personal mobile device with a passcode.
+> [!IMPORTANT]
+> Only choose to use this option on a device you know to be secure. A secure device may be one that requires you to log in before being able to access the Internet, for example one within an office network, or a personal mobile device with a passcode.
 
 **A device shared between multiple people would not be considered secure and you should not use this option.**
 </div>
+```
```

Comment:
These backticks look like they might be a mistake? And the `</div>` should be removed
```suggestion
```

Reply:
Updated